fastly/data_source_fastly_tls_private_key_ids.go

```diff
@@ -0,0 +1,47 @@
+package fastly
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/hashcode"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceTLSPrivateKeyIDs() *schema.Resource {
```

Comment:
Should this not be named consistently with the other data sources?
```suggestion
func dataSourceFastlyTLSPrivateKeyIDs() *schema.Resource {
```

I presume we'd want to update the READ function as well: 

```diff
- dataSourceTLSPrivateKeyIdsRead
+ dataSourceFastlyTLSPrivateKeyIdsRead
```